api/data_explorer/swagger/swagger.yaml

```diff
@@ -17,8 +17,32 @@ paths:
       parameters: []
       responses:
         200:
-          description: "Facets."
+          description: "Success"
           schema:
-            type: "string"
+            $ref: "#/definitions/FacetsResponse"
       x-swagger-router-controller: "data_explorer.controllers.facets_controller"
-definitions: {}
+definitions:
+  FacetsResponse:
```

Comment:
This definition is a bit redundant. I'm not an expert on API definition but consider removing this and then changing the /facets 200 response to something like:

```
description: "Success"
schema:
  type: array
  items:
    $ref: "#/definitions/Facet"
```